build.sbt

```
@@ -30,7 +27,8 @@ lazy val assembler = project
 lazy val cli = project
   .settings(
     name := "Command line interface",
-    libraryDependencies ++= cliDependencies
+    libraryDependencies ++= cliDependencies,
+    assemblyJarName in assembly := "geney-" + version.value + ".jar"
```

В `scala` вместо конкатенации принято использовать интерполированные строки:
```suggestion
    assemblyJarName in assembly := s"geney-${version.value}.jar"
```